compiler/test/fail_compilation/templatethis.d

```diff
@@ -0,0 +1,52 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/templatethis.d(11): Error: cannot use `this` outside an aggregate type
+fail_compilation/templatethis.d(15): Error: cannot use `this` outside an aggregate type
+fail_compilation/templatethis.d(19): Error: cannot use `this` outside an aggregate type
+fail_compilation/templatethis.d(21): Error: cannot use `this` outside an aggregate type
+---
+*/
+
```

Comment:
Use `#line` for stable line numbers.

Reply:
Well, then the error message line numbers don't match up.

Reply:
That is the entire point, adding/removing a line doesn't affect the entire output.

Reply:
Using ` #line` is also annoying since the error line that is reported does not match up the editor line and you can't just `:N`